blocks/cards/cards.js

```diff
@@ -10,7 +10,29 @@ export default function decorate(block) {
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
       if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      else { div.className = 'cards-card-body'; }
+
+      /* cards simple */
+      if ( block.classList.contains('simple')) {
+        const anchorEl = div.querySelector('a');
+        if (anchorEl) {
```

Comment:
Can you explain this conditional? anchorEl will always be true here

Reply:
Anchor element is not mandatory for simple cards. We may or may not have the card linked to a url.

Reply:
Could you implement it with a single conditional though? It would make it a bit more readable.